microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -22,145 +22,168 @@ ms.date: 05/04/2021
+If you run into issues while using workspaces with your framework of choice, or you can't get it working after a custom configuration, [start a thread in the mailing list](https://groups.alphabet.com/forum/#!forum/alphabet-browser-developer-tools) or [ask a question on Stack Overflow](https://stackoverflow.com/questions/ask?tags=alphabet-browser-devtools) to exchange information with the rest of the DevTools community.
 
 
 <!-- ====================================================================== -->
-## Related feature: Local overrides
+#### Related feature: Overrides
+
+**Overrides** is a DevTools feature that’s similar to a workspace. You can use an override when you want to experiment with changes to a webpage, and you need to display the changes across webpage loads, but you don't care about mapping your changes to the source code of the webpage. However, your changes aren’t saved when you refresh the webpage. 
 
-**Local Overrides** is another DevTools feature that is similar to Workspaces.  Use Local Overrides when you want to experiment with changes to a webpage, and you need to display the changes across webpage loads, but you don't care about mapping your changes to the source code of the webpage.
+The **Overrides** feature lets you store a local copy of the webpage files on the server. When you refresh the page, Microsoft Edge loads the local copy of files instead of the files on the server. To learn more about overrides, see [Override webpage resources with local copies (Overrides tab)](../javascript/overrides.md).
 
 <!--Todo: add section when content is ready  -->
 
 
 <!-- ====================================================================== -->
-## Step 1: Set up
 
-We'll set up the demo and then set up DevTools.
+## Set up the environment for the tutorial
+
+We'll set up the demo files, and then set up DevTools.
 
-### Set up the demo
+#### Set up the demo
 
-1. [Open the demo source code](https://github.com/MicrosoftEdge/Demos/tree/main/workspaces).
+1. [Get the workspaces demo source code](https://github.com/MicrosoftEdge/Demos/tree/main/workspaces).
 
-1. Create an `app` directory on your desktop.  Save copies of the `index.html`, `styles.css`, and `script.js` files from the demo source code to the `app` directory.  For the rest of the tutorial, the directory is referred to as `~/Desktop/app`.
+1. Create a `app` directory on your desktop.  Save copies of the `index.html`, `styles.css`, and `script.js` files from the demo source code to the `app` directory.  For the rest of the tutorial, this directory is referred to as `~/Desktop/app`. <!-- Michael: I don't know the rationale behind using the ~/Desktop/app construct.   --> 
```

Comment:
```suggestion
1. Create an `app` directory on your desktop.  Save copies of the `index.html`, `styles.css`, and `script.js` files from the demo source code to the `app` directory.  For the rest of the tutorial, this directory is referred to as `~/Desktop/app`, though you can use a different path.
```
When I doc'd Workspaces in another article, I set up `~/Desktop/app` and had success.